index.html

```diff
@@ -33,9 +33,149 @@
      <a href="https://www.google.com"><i class="fa fa-google"></i></a>
     </li>
     </ul>
+   
+        <img class="seebutton" src="./Disabled.png" alt="button">
+    
 </div>
 </section>
-
+<section>
+    <div class="second-header">
+        <h2 class="second">My Recent work</h2>
+        <img src="./Indicator.png" alt="indicator">
+    </div>
+    <div class="grid-container">
+        <div class="grid-item first">
+            
+           <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+           </div>
+       
+        <div class="grid-item third">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fourth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+        <div class="tagcontainer">
+         <label>Ruby on Rails</label>
+         <label>Javascript</label>
+         <label>CSS</label>
+         <label>HTML</label>
+        </div>
```

Comment:
- [ ] below is the project requirement please make sure to make use of `ul` & `li` tags to implements these requirements.  therefore rather than having label these are the tags to be used :+1: 
![req-full](https://user-images.githubusercontent.com/89055502/235299538-dc22aa64-a765-446c-8309-42e5579eb507.png)